### ISSUE TITLE
No Adapter From Version $14 for Mul

### ISSUE THREAD

**Issue by roachsinai** [onnx/onnx#6297]
# Bug Report

### Is the issue related to model conversion?

Yes.

### Describe the bug

<img width="1481" alt="image" src="https://github.com/user-attachments/assets/b9234fd4-12f8-40b9-a84d-389f71c86d76">

**Comment by justinchuby** [comment]
Backward conversion is not complete. You may register it here https://github.com/onnx/onnx/blob/99e1fd352c05c3176770080824fd7a8c474c97c0/onnx/version_converter/convert.h#L511 and send a PR. Thanks.

**Comment by gramalingam** [comment]
It should be a type restriction, so doable.

**Comment by roachsinai** [comment]
Yes, I've tried register `Mul` , it works for `Mul`. But then got same error for another operator that have not been registered.
